components/navbar/UserBubble.js

```diff
@@ -9,12 +8,44 @@ import {
   Button,
   Tooltip,
   MenuItem,
+  Skeleton,
 } from "@mui/material";
-import { useOperationMethod } from "react-openapi-client";
+import useCurrentUser from "../../modules/authentication/hooks/use-current-user";
+import useAuthError from "../../modules/authentication/hooks/use-error";
+import useAuthLoading from "../../modules/authentication/hooks/use-loading";
+import useSignOut from "../../modules/authentication/hooks/use-sign-out";
 
-function UserBubble({ sessionActions, userName, userId }) {
+function UserBubble() {
   const [anchorElUser, setAnchorElUser] = React.useState(null);
```

Comment:
Could be better to import explicitly the useState method?

Reply:
could be, this is just the way it was in the component source code:
<img width="638" alt="image" src="https://user-images.githubusercontent.com/73831187/193964168-5ccc2246-d6df-41c7-a414-2ed520f0734c.png">
check it on: https://mui.com/material-ui/react-app-bar/

Reply:
I understand. I would ask a mentor as you said. Because I think it is weird to import explicitly React when using NextJs